components/InputField.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+interface InputProps {
```

Comment:
why is everything optional?

Reply:
In the inputfield component, there are different types to be passed through. I used the input component in the form that accepts email and password and also used the same input component for the checkbox.

Reply:
yes but all the props you have are optional, when input component is used, it is possible to just call the component element and not pass anything, the component will not make sense right? or it will not have a use